pkg/reconciler/eventlistener/resources/container.go

```diff
@@ -56,16 +56,18 @@ func MakeContainer(el *v1beta1.EventListener, configAcc reconcilersource.ConfigA
 			Capabilities: &corev1.Capabilities{
 				Drop: []corev1.Capability{"ALL"},
 			},
-			// 65532 is the distroless nonroot user ID
-			RunAsUser:    ptr.Int64(65532),
-			RunAsGroup:   ptr.Int64(65532),
 			RunAsNonRoot: ptr.Bool(true),
 			SeccompProfile: &corev1.SeccompProfile{
 				Type: corev1.SeccompProfileTypeRuntimeDefault,
 			},
 		}
 	}
 
+	if *c.SetSecurityContextIDs {
```

Comment:
@dibyom @khrm

I have few query
* should we actually maintain 2 flags `el-security-context` and  `el-security-context-run-as-ids`
* Is there any usecase where someone don't want to set securityContext for EventListener??
* If its okay to set securityContext by default on EL then planning to use just one flag `el-security-context-run-as-ids` to set runAsUser and runAsGroup as these are not needed for Openshift

Do let me know your thoughts

**Note**: if i change flag from `el-security-context` **--->**  `el-security-context-run-as-ids the upgrade is success

Reply:
For Openshift you  want to set the all fields of the default security context except for the RunAsUser and RunAsGroup? Maybe instead of adding another flag, we add read the default values of RunAsUser and RunAsGroup from [`config-defaults`](https://github.com/tektoncd/triggers/blob/main/config/config-defaults-triggers.yaml#L42) or the config-features config maps? We can set the default to the current value and for Openshift, you can override it to another value or set it to empty?

Reply:
Yes make sense
So that way user can configure whatever value they want for runAsUser and runAsGroup :+1: 

Will change and update the PR 
Thank you

Reply:
Updated PR
PTAL 
Thank you

Reply:
This is a better approach. :)